docs/runbooks/VirtControllerDown.md

```diff
@@ -1,40 +1,49 @@
+<!-- Edited by Jiří Herrmann, 9 Nov 2022 -->
+
 # VirtControllerDown
 
 ## Meaning
-No running virt-controller was detected for the last 5 min. Virt-controller deployment has a default replica of 2 pods.
+No running `virt-controller` pod has been detected for 5 minutes.
 
 ## Impact
-Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+Any actions related to virtual machine (VM) lifecycle management fail. This notably includes launching a new VM instance (VMI) or shutting down an existing VMI.
 
 ## Diagnosis
 
-- Set the environment variable NAMESPACE
+1. Set the `NAMESPACE` environment variable:
+```bash
+$ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+```
 
-    ```
-    export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
-    ```
+2. Check the status of the `virt-controller` deployment:
+```bash
+$ kubectl get deployment -n $NAMESPACE virt-controller -o yaml
+```
 
-- Observe the status of the virt-controller deployment
-    ```
-    kubectl get deployment -n $NAMESPACE virt-controller -o yaml
-    ```
-- Observe the logs of the kubecontroller manager pod, to see why it cannot create the virt-controller pods.
-    ```
-    kubectl get logs <virt-controller-pod>
-    ```
+3. Review the logs of the `kubecontroller` manager pod, to see why it cannot create the `virt-controller` pods.
+```bash
+$ kubectl get logs <virt-controller-pod>
+```
 
-    *Note:* virt-controller pod should have name similar to virt-controller-7888c64d66-dzc9p.\
-    There could be several pods that run virt-controller
+Note that the `virt-controller` pod must have a name similar to `virt-controller-7888c64d66-dzc9p.\`.
+In addition, several different pods might be running `virt-controller`.
```

Comment:
@sradco What does this note mean? That the user needs to check each virt-controller pod log? 

If yes, then I recommend removing the note because I think that would be obvious to the user.

Reply:
I think it about the number of possible replicas. @enp0s3 am I correct?